### ISSUE TITLE
In Select topic dialog; Fold All only closes the root node

### ISSUE THREAD

**Issue by netmackan** [raydac/netbeans-mmd-plugin#10]
Way to reproduce:
1. In a mind map with multiple nodes containing child nodes: right click on a node
2. Select "Add jump"
3. The "Select topic" dialog opens with all nodes unfolded. Click Fold All.
4. It seems all nodes are folded as the root node is folded. But now click the "+" to expand the root node.
Expected: Now only the direct child nodes of the root should be visible.
Actual: The whole tree is unfolded

Basically it seems the "Fold All" toolbar button only closes the root node.
